src/logic/operate.js

```diff
@@ -0,0 +1,28 @@
+import Big from 'big.js';
+
+function Operate(numberOne, numberTwo, operation) {
+  const numOne = Big(numberOne);
+  const numTwo = Big(numberTwo);
+
+  let valueArg;
+
+  if (operation === 'X') {
+    valueArg = numOne.times(numTwo);
+  }
+  if (operation === '-') {
+    valueArg = numOne.minus(numTwo);
+  }
+  if (operation === '+') {
+    valueArg = numOne.plus(numTwo);
+  }
+  if (operation === '÷') {
+    valueArg = numOne.div(numTwo);
+  }
+  if (operation === '%') {
+    valueArg = numOne.mod(numTwo);
+  }
```

Comment:
- [ ] Since the percentage operator(%) has already been handled correctly in calculate.js, please remove this conditional case here.